koocook_core/tests/test_support/test_markdown.py

```diff
@@ -4,12 +4,86 @@
 from ...support.markdown import MarkdownSource
 
 
-# TODO: Preliminary tests
-class TestMarkdown(unittest.TestCase):
-    def test_proper_escape(self):
+def load_article():
+    import json, os
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    article = json.loads(open(os.path.join(dir_path, "test_markdown/article.json")).read())
+    return article
+
+
+class TestMarkdownSource(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.text = "This is a no markdown text"
+
+    def test_no_markdown_characters(self):
+        no_md_text = "*"
+        source = MarkdownSource(no_md_text)
+        self.assertEqual(source.rendered, f"<p>{no_md_text}</p>")
+        no_md_text = "`"
+        source = MarkdownSource(no_md_text)
+        self.assertEqual(source.rendered, f"<p>{no_md_text}</p>")
```

Comment:
Testing this way, once one test fails, the other doesn't even run. I suggest using `subTest`s